src/cljs/ataru/hakija/application_handlers.cljs

```diff
@@ -136,12 +135,17 @@
                    (merge answer {:value new-value
                                   :valid valid?}))))))
 
+(defn- toggle-values
+  [answer options]
+  (reduce (fn [answer option-value]
```

Comment:
Can this be `(reduce toggle-multiple-choice-option answer options)` or is the `nil` parameter required?